MobileBuy/sample/src/shopify/java/com/shopify/sample/data/graphql/Converter.java

```diff
@@ -0,0 +1,46 @@
+package com.shopify.sample.data.graphql;
+
+import com.shopify.buy3.Storefront;
+import com.shopify.sample.domain.model.Collection;
+import com.shopify.sample.domain.model.Product;
+import com.shopify.sample.util.Util;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+
+public final class Converter {
```

Comment:
private constructor

Reply:
It is fixed but GitHub does not know it.